pkg/querier/querier.go

```diff
@@ -40,6 +43,15 @@ type AgentNetworkPolicyInfoQuerier interface {
 	GetRuleByFlowID(ruleFlowID uint32) *types.PolicyRule
 }
 
+type AgentMulticastInfoQuerier interface {
+	// GetIGMPReportNPStats gets statistics generated by networkpolicies that block or allow
+	// IGMP reports messages.
+	GetIGMPReportNPStats() map[cpv1beta.NetworkPolicyType]map[apitypes.UID]map[string]*types.RuleMetric
+	// GetGroupPods gets a map which encode which Pods have joined group for local Node.
```

Comment:
"a map that saves the local Pod members of multicast groups on the Node"

Reply:
updated.